helpdesk/helpdesk/templatetags/filter_helpers.py

```diff
@@ -0,0 +1,36 @@
+from typing import Any
+
+from django import template
+from django.forms import Field
+from django_filters.fields import ChoiceField
+
+register = template.Library()
+
+
+def _get_value_display_for_choices(choice_field: ChoiceField, value: Any) -> Any:
+    choices = dict(choice_field.choices.choices)
+    return choices.get(value, value)
+
+
+def _get_value_display_for_field(field: Field, value: Any) -> Any:
+    if isinstance(field, ChoiceField):
+        return _get_value_display_for_choices(field, value)
+
+    return value
+
+
+@register.inclusion_tag("inc/tags/filter_badges.html", takes_context=True)
+def render_filter_badges(context: dict[str, Any]) -> dict[str, Any]:
+    fltr = context["filter"]
```

Comment:
This feels a bit implicit -- perhaps we should pass this in explicitly?

Reply:
3f454f1